pages/login.html

```diff
@@ -497,6 +497,9 @@ <h3 class="separator"><span>or</span></h3>
                       placeholder="Type the sign in code we emailed you"
                       value=""
                       autocomplete="off"
+                      inputmode="numeric"
+                      pattern="[0-9]*"
+                      type="text"
```

Comment:
```suggestion
                      type="number"
```
Wouldn't this be more appropriate?

Reply:
No, as that will show the increment/decrement arrows which may lead to user mistakenly changing the entered OTP.

Reply:
Yeah, fair point.  Turns out you can hide those arrows, but it still responds to up or down keypresses by incrementing or decrementing.